Experimental/src/org/sleuthkit/autopsy/experimental/autoingest/AutoIngestCaseManager.java

```diff
@@ -49,21 +59,81 @@ synchronized static AutoIngestCaseManager getInstance() {
      * Constructs an object that handles locating and opening cases created by
      * auto ingest.
      */
-    private AutoIngestCaseManager() {
+    private AutoIngestCaseManager() throws AutoIngestCaseManagerException {
+        try {
+            init();
+        } catch (AutoIngestCaseManager.AutoIngestCaseManagerException ex) {
+            Logger.getLogger(AutoIngestCaseManager.class.getName()).log(Level.SEVERE, null, ex);
+            throw ex;
+        }
+    }
+    
+    /**
+     * Initialize the coordination service.
+     * 
+     * @throws org.sleuthkit.autopsy.experimental.autoingest.AutoIngestCaseManager.AutoIngestCaseManagerException 
+     */
+    private void init() throws AutoIngestCaseManager.AutoIngestCaseManagerException {
+        try {
+            coordinationService = CoordinationService.getInstance();
+        } catch (CoordinationServiceException ex) {
+            throw new AutoIngestCaseManager.AutoIngestCaseManagerException(ex.getMessage(), ex);
```

Comment:
The AutoIngestCaseManager should add some context to the Coordination service message, the message of which will already be logged in the stack trace, since the Autopsy Logger recurses through nested exceptions. Please replace the ex.getMessage with a suitable message created via an @Messages annotation, e.g., "Failed to get coordination service." The client can then add additional context when it logs the exception.